packages/react/src/components/Dropdown/Dropdown.stories.js

```diff
@@ -57,7 +57,83 @@ const items = [
   },
 ];
 
-export const Default = (args) => (
+export const Playground = (args) => (
+  <div style={{ width: 400 }}>
+    <Dropdown
+      {...args}
+      id="default"
+      titleText="Dropdown label"
+      helperText="This is some helper text"
+      label="Dropdown menu options"
+      items={items}
+      itemToString={(item) => (item ? item.text : '')}
+    />
```

Comment:
Args need to be spread last so we can change the defined props with the controls

```suggestion
    <Dropdown
      id="default"
      titleText="Dropdown label"
      helperText="This is some helper text"
      label="Dropdown menu options"
      items={items}
      itemToString={(item) => (item ? item.text : '')}
      {...args}
    />
```